modules/signals/src/index.ts

```diff
@@ -1,11 +1,14 @@
 export { DeepSignal } from './deep-signal';
-export { getState } from './get-state';
-export { PartialStateUpdater, patchState } from './patch-state';
-export { signalState } from './signal-state';
+export { signalState, SignalState } from './signal-state';
 export { signalStore } from './signal-store';
 export { signalStoreFeature, type } from './signal-store-feature';
 export { SignalStoreFeature } from './signal-store-models';
-export { StateSignal } from './state-signal';
+export {
+  getState,
+  PartialStateUpdater,
+  patchState,
+  StateSource,
```

Comment:
Unrelated to the PR, but do we want to have migration in place to update StateSignal to StateSource ?

Reply:
yes, that would be great 👍